.github/workflows/publish.yml

```
@@ -124,7 +123,7 @@ jobs:
             "Apply Run from GitHub Actions CI ${{ inputs.tag || github.sha }}"
 
   publish-images-to-ghcr:
-    if: ${{ github.ref == 'refs/heads/main' && github.event_name == 'release' }}
+    if: ${{ github.ref == 'refs/heads/main' || github.event_name == 'release' }}
```

We don't publish to ghcr on manual actions - they are to push hotfixes to our infra.
```suggestion
    if: ${{ github.event_name == 'release' }}
```